woocommerce/wc-admin
# Remove HomeScreenFeedback note

Fixes #5938 

This PR removes HomeScreenFeedback.php note and its references.

### Detailed test instructions:

1. Empty `wp_wc_admin_notes` table.
2. Navigate to WordPress -> Plugins and deactivate WooCommerce admin then activate it again
3. Check `wp_wc_admin_notes` table and confirm that there is no record of HomeScreenFeedback
